### PR TITLE
Car port: LEXUS UX 2019

### PR DESCRIPTION
**Pull request by brianhaugen2**
**Checklist**
- [ x ] added to README
- [ x ] test route added to [test_routes.py](../../selfdrive/test/test_routes.py)
- [ x ] route with openpilot: [https://my.comma.ai/f33ec5db3dc94da9/1621523379300/1621524882825](https://my.comma.ai/f33ec5db3dc94da9/1621523379300/1621524882825) 
- [ x ] route with stock system: [https://my.comma.ai/f33ec5db3dc94da9/1621520659477/1621520788918](https://my.comma.ai/f33ec5db3dc94da9/1621520659477/1621520788918)

### CODE REVIEW

File: selfdrive/test/test_routes.py [hunk 1]
```diff
@@ -371,6 +371,11 @@
     'enableCamera': True,
     'enableDsu': False,
   },
+  "f33ec5db3dc94da9|2021-05-20--09-09-39": {
```
Comment (adeebshihadeh) [inline selfdrive/test/test_routes.py:374]: this route doesn't have the full logs uploaded. connect your device to wifi and enable the "Upload Raw Logs" toggle until the route fully uploads

File: selfdrive/car/toyota/values.py [hunk 2]
```diff
@@ -292,6 +293,9 @@ class CAR:
...
+  }],
+  CAR.LEXUS_UXH_TSS2: [{
```
Comment (adeebshihadeh) [inline selfdrive/car/toyota/values.py:297]: add CAR.LEXUS_UXH_TSS2 to IGNORED_FINGERPRINTS

File: selfdrive/test/test_routes.py (hunk 1 above)
Comment (brianhaugen2) [reply selfdrive/test/test_routes.py]: Changed the test route to the latest on comma and awaiting upload to complete

File: selfdrive/test/test_routes.py [hunk 3]
```diff
@@ -371,6 +371,11 @@
     'enableCamera': True,
     'enableDsu': False,
   },
+  "f33ec5db3dc94da9|2021-05-22--18-19-45": {
+    'carFingerprint': TOYOTA.LEXUS_UXH_TSS2,
+    'enableCamera': True,
+    'enableDsu': False,
```
Comment (adeebshihadeh) [inline selfdrive/test/test_routes.py:377]:
```suggestion
```

**Pull request merged**